CHANGELOG.md

```diff
@@ -0,0 +1,53 @@
+Changelog
+=========
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/) and this project adheres to
+[Semantic Versioning](https://semver.org/).
+
+But we group the changes as follows:
+
+* *Breaking Changes* when a change requires manual activity at update.
+* *Bug Fixes* when we fix broken functionality.
+* *New Features* for added functionality.
+* *Improvements* for other changed parts.
+
+
+Table of Contents
+-----------------
+
+[TOC]
+
+
+[Unreleased]
+------------
+
+### Breaking Changes
+
+### Bug Fixes
+
+### New Features
+
+### Improvements
+
+
+--------------------------------------------------------------------------------
+
+
+[1.5.0] (2019-09-12)
+--------------------
+
+### Improvements
+
+* Check now also individual `WebElement`s, making recheck a "Jest for Java".
```

Comment:
Hmmm, snapshot testing is just a small part of Jest. It also has a test runner, mocking capabilities, coverage tracker, … moreover, difference testing is different than snapshot testing as we don't diff text. You really wanna keep it? I find it a bit confusing.

Reply:
If someone enters "Jest for Java" as a search term, I want recheck to show up. There are differences, but still... I think in the Changelog, it doesn't hurt...

Reply:
Hmmm, I don't think that the changelog is the right place to do SEO … here we want to give clear information to our users what has changed, no marketing blabber.

Just my two cents, feel free to merge anyway.